sketchlib/ArcAngles.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Angles for describing a circular arc
+ */
+export class ArcAngles {
```

Comment:
This class needs to be fleshed out more. I like the way I did it in `math-notebook`, see https://github.com/ptrgags/math-notebook/blob/main/mobius/src/geometry/arc_angles.rs, albeit be a little less strict. Allow an angle of 0 vs an angle of 2pi